### DISCUSSION TITLE
Help - Not able to get the plugin working

### DISCUSSION THREAD

**Discussion by patppuccin** [tim-hub/obsidian-bible-reference#9] [General]
Hi, I am not sure how to get the plugin running. I installed it and enabled it. But unable to get the reference suggestions when I type `--john1:1`. Does the plugin have a video walk through or something?

Was not sure if this is where I should ask this by the way.
Thanks in advance for the response.

**Comment by tim-hub** [comment]
@Patrick-Ambrose thanks for the feedback, may I know 

- which OS and which version of Obsidian are you using? 
- do you mind sharing your obsidian bible reference plugin setting (screenshot to me)? 

:)

**Reply by patppuccin** [reply]
![Screenshot from 2021-12-13 07-22-41](https://user-images.githubusercontent.com/61621587/145740560-e18cd685-e675-44fc-8648-864c2d309359.png)
Linux (PopOS) and Obsidian v0.12.19

**Reply by tim-hub** [reply]
Thanks, will have a look, I may try in a VM to see if anything I can find

**Reply by tim-hub** [reply]
Thanks again

**Reply by patppuccin** [reply]
Great! Thanks for the quick reply.

**Reply by Valenalsorna** [reply]
Hello Patrick
I couldn't get it to work on Android. I have removed it off my macboor air and then reinstalled it. Unfortunately I can't get it to work.I have selected NKJV


Robert

**Reply by tim-hub** [reply]
Hi guys, sorry for the delay.

I haven't gotten a chance to look it yet.

I will have a look tonight, it might due to the api proxy

**Reply by tim-hub** [reply]
From what I can tell on my side, it seems like related to the network.

I could not use the plugin, when my android firewall is on, after allow obsidian internet access, the plugin works well. 


😢

**Reply by DouglasDahlia779** [reply]
the plugin works on my ipad.

**Comment by tim-hub** [comment]
Hi @Patrick-Ambrose 

I found the issue now, the api, bible-api.com is down now, I think I need to host my own one now. 

https://github.com/seven1m/bible_api/issues/60

I don't think I could fix it today, but I will update you when we can switch to new api instance. 

Thanks

**Reply by patppuccin** [reply]
Thanks!

**Reply by tim-hub** [reply]
> Thanks!
> 

Bible-api.com is back now.

Let me know if u still cannot use it 😂

**Reply by patppuccin** [reply]
Yes, it works now!

Thanks.

**Reply by billpetro** [reply]
Ah, I found my mistake. It's not `--john 1:1` it's `john1:1` with no spaces.

**Reply by Valenalsorna** [reply]
Hello guys
I have just installed an enabled plug in. Tried typing in a new note but no reference coming up on screen. I’m using IPad os 15

Cheers

**Reply by tim-hub** [reply]
> Hello guys I have just installed an enabled plug in. Tried typing in a new note but no reference coming up on screen. I’m using IPad os 15
> 
> Cheers

Hi @Valenalsorna 

which obsidian version are u using? 

BTW, Cause I don't have an IOS device, can you help me that to try it on any other devices? to tell whether it is broken on IOS only.

Thanks :)

**Reply by Valenalsorna** [reply]
Hello
I am running Obsidian version 1.2.0(50)
I have a pixel book. Do you want me to try on Android?

Regards
Robert

**Reply by tim-hub** [reply]
> Hello I am running Obsidian version 1.2.0(50) I have a pixel book. Do you want me to try on Android?
> 
> Regards Robert

Yeah, that will be great,

How was it going on android?

**Comment by mjh8431** [comment]
I am having the same problem. I am using Mac OS 12.3 (Monterey) and Obsidian 0.14.6. It had been working until I tried it today. I tied disabling/enabling - no effect; I also uninstalled/reinstalled - again no effect. I tried changing the Bible version - no effect.

Thanks for any help you can give.

**Reply by tim-hub** [reply]
Thanks for the feedback.

One bible api provider was blocked. from bible-api.com

NIV: New International Version, 1984 (Bolls Life)
NKJV: New King James Version (Bolls Life)
CUV: China Union Version (Bolls Life)
ESV

These 4 versions suppose to work.

---

I will have a look of bible-api.com,

I might need to re-host this api, or build a proxy for it to make it work again.

**Reply by tim-hub** [reply]
I just created a ticket.

https://github.com/tim-hub/obsidian-bible-reference/issues/32

**Reply by tim-hub** [reply]
bible-api.com should be back to normal now

**Reply by tim-hub** [reply]
so the plugin should work again

**Reply by Valenalsorna** [reply]
Tim-Hub
Appreciate all your work, however i still can’t get it to work. A real shame

**Reply by tim-hub** [reply]
> Tim-Hub
> Appreciate all your work, however i still can’t get it to work. A real shame

Is it not working on your iPad or android?
Or all devices u have? Is it possible related to network settings?

On my android device it seems fine, but I don't have an iPad.

**Comment by mjh8431** [comment]
Thanks


———————————
Martin Harris
***@***.***

**Comment by Valenalsorna** [comment]
Hi Guys
A quick update. It seems to be working on my M1 mac ok. Haven't tried IPAD
yet. Thanks for all your help and hard work
Blessings
Rob
*robert snoddy*

**Comment by nathanwinklepleck** [comment]
I'm also running into issues with the ESV version. I'm running the latest Obsidian version on an M1 Max MacBook Pro. Version 0.15.9 (installer 0.15.8)

When I type "--Obe" nothing comes up nor do I get anything with "--John".

**Reply by tim-hub** [reply]
Can you try enable developer console? @nathanwinklepleck 

![image](https://github.com/tim-hub/obsidian-bible-reference/assets/2884320/f0b63c42-f9ff-450e-9ada-b5cc9a79023b)


If you can bring some logs, that will be very helpful for debugging
